### PR TITLE
(feat) adding litmus support

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: spec/spec_helper_acceptance_local.rb [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+def create_and_upload_certs
+  cert_files = ['privkey.pem', 'ca.pem', 'ca2.pem', 'chain.pem', 'chain2.pem', 'leafkey.pem', 'leaf.pem', 'leafchain.pem', 'leafchain2.pem', 'leaf.p12', 'leaf2.p12']
+  recreate_certs = false
+  cert_files.each do |cert_file|
+    recreate_certs = true unless File.file?("spec/acceptance/certs/#{cert_file}")
+  end
+  create_certs if recreate_certs
+  cert_files.each do |cert_file|
+    if ENV['TARGET_HOST'].nil? || ENV['TARGET_HOST'] == 'localhost'
+      Open3.capture3("cp spec\\acceptance\\certs\\#{cert_file} C:\\tmp\\#{cert_file}")
```
Comment (michaeltlombardi) [inline spec/spec_helper_acceptance_local.rb]: I think this is the line causing failures on Windows, Open3 defaults to cmd which does not have a `cp` command, we would need to use `copy` or drop into PowerShell.

This line also assumes the only time we're doing localhost testing is on a Windows machine?

File: provision.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  images: ['waffleimage/debian8']
+travis_deb:
+  provisioner: docker
+  images: ['debian:8', 'debian:9', 'ubuntu:14.04', 'ubuntu:16.04', 'ubuntu:18.04']
+travis_el:
+  provisioner: docker_exp
+  images: ['centos:6', 'centos:7', 'oraclelinux:6', 'oraclelinux:7', 'scientificlinux/sl:6', 'scientificlinux/sl:7']
+release_checks:
+  provisioner: vmpooler
+  images: ['redhat-6-x86_64', 'redhat-7-x86_64', 'redhat-8-x86_64', 'centos-6-x86_64', 'centos-7-x86_64', 'oracle-6-x86_64', 'oracle-7-x86_64', 'scientific-6-x86_64', 'scientific-7-x86_64', 'debian-8-x86_64', 'debian-9-x86_64', 'debian-10-x86_64',  'ubuntu-1404-x86_64', 'ubuntu-1604-x86_64', 'ubuntu-1804-x86_64', 'sles-11-x86_64', 'sles-12-x86_64', 'sles-15-x86_64', 'win-2008r2-x86_64', 'win-2012r2-x86_64', 'win-2016-x86_64', 'win-10-pro-x86_64']
```
Comment (lionce) [inline provision.yaml]: hey @michaeltlombardi ,

Comparing with metadata, redhat5, centos5, oracle5 are missing. Also sles, for windows some versions and solaris. Do we need to add those platforms also for testing or to update metadata 🤔 ?

Cheers

Comment (lionce) [reply provision.yaml]: would you also add a screenshot with the successful test report?

Comment (lionce) [reply provision.yaml]: PIpelines is failing
https://pipelines.puppet.com/team-modules/builds/603037

**Comment by michaeltlombardi** [comment]
Okay! Passing locally, in appveyor, in travis, and also in Pipelines!

https://pipelines.puppet.com/team-modules/builds/604064

**Pull request merged**